docs/database/database-spans.md

```diff
@@ -268,6 +302,98 @@ Placeholders in a parameterized query SHOULD not be sanitized. E.g. `where id =
 e.g. from `IN (?, ?, ?, ?)` to `IN (?)`, as this can help with extremely long IN-clauses,
 and can help control cardinality for users who choose to (optionally) add `db.query.text` to their metric attributes.
 
+## Generating a summary of the query text
+
+The `db.query.summary` attribute captures a shortened representation of a query text
+which SHOULD have low-cardinality and SHOULD NOT contain any dynamic or sensitive data.
+
+> [!NOTE]
+> The `db.query.text` attribute is intended to identify individual queries. Even though
+> it is sanitized if captured by default, it could still have high cardinality and
+> might reach hundreds of lines of code.
+>
+> The `db.query.summary` is intended to provide a less granular grouping key that
+> can be used as a span name or a metric attribute in common cases. It SHOULD
+> only contain information that has a significant impact on the query, database,
+> or application performance.
+
+Instrumentations that support query parsing SHOULD generate query summary when
+one is not readily available from other sources.
+
+The summary SHOULD preserve the following parts of query in the order they were provided:
+
+- operations such as SQL SELECT, INSERT, UPDATE, DELETE or other commands
+- operation targets such as collection or database names
+
+Instrumentation MAY include additional details such as specific SQL clauses as long
+as summary remains relatively short and its cardinality remains low comparing to
+the `db.query.text`.
```

Comment:
any particular SQL clauses you have in mind?

Reply:
I had `JOIN` in mind, but nothing else. I'm considering to remove this sentence - it doesn't seem to convey anything new.

Reply:
moved to the end of the section:

> Semantic conventions for individual database systems or specialized instrumentations
MAY specify a different `db.query.summary` format as long as produced summary remains
relatively short and its cardinality remains low comparing to the `db.query.text`.